views/js/controller/TaoProctoring/report.js

```diff
@@ -112,6 +112,7 @@ define([
                 })
                 .datatable({
                     url: serviceUrl,
+                    data: dataset,
```

Comment:
Woops, this won't work since the new version of the datatable: the dataset must be provided in the second argument, not in the config